source/_integrations/duke_energy.markdown

```diff
@@ -0,0 +1,33 @@
+---
+title: Duke Energy
+description: Instructions on how to integrate Duke Energy within Home Assistant.
+ha_category:
+  - Energy
+ha_release: 2024.10
+ha_iot_class: Cloud Polling
+ha_codeowners:
+  - '@hunterjm'
+ha_domain: duke_energy
+ha_config_flow: true
+ha_integration_type: integration
+---
+
+The Duke Energy integration allows you to get energy information from [Duke Energy](https://www.duke-energy.com/). Currently, this integration only supports electric meters.
+
+## Authentication Requirements
+
+Only a username and password are required to access your account.
+
+## Energy
+
+Because Duke Energy only releases usage data with around a 48-hour delay, the integration inserts data into statistic objects.
+You can find the statistics in {% my developer_statistics title="**Developer Tools** > **Statistics**"%} and search for "duke_energy".
```

Comment:
```suggestion
To view the statistics, go to {% my developer_statistics title="**Developer Tools** > **Statistics**"%} and search for "duke_energy".
```

Reply:
That page doesn’t let you view the data. You must add it to the energy dashboard for that.